azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/utilities/BottomCellViewHolder.kt

```diff
@@ -4,89 +4,30 @@
 package com.azure.android.communication.ui.utilities
 
 import android.view.View
-import android.widget.ImageView
+import android.view.accessibility.AccessibilityEvent
+import android.view.accessibility.AccessibilityNodeInfo
 import android.widget.TextView
-import androidx.core.content.ContextCompat
-import androidx.core.view.AccessibilityDelegateCompat
-import androidx.core.view.ViewCompat
-import androidx.core.view.accessibility.AccessibilityNodeInfoCompat
 import androidx.recyclerview.widget.RecyclerView
 import com.azure.android.communication.ui.R
-import com.microsoft.fluentui.persona.AvatarView
 
-internal class BottomCellViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
-    private val imageView: ImageView = itemView.findViewById(R.id.cell_icon)
-    private val avatarView: AvatarView =
-        itemView.findViewById(R.id.azure_communication_ui_participant_list_avatar)
+internal open class BottomCellViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
     private val title: TextView = itemView.findViewById(R.id.cell_text)
-    private val accessoryImage: ImageView = itemView.findViewById(R.id.cell_check_mark)
-    private var onClickAction: Runnable? = null
-
-    init {
-        itemView.setOnClickListener {
-            onClickAction?.run()
-        }
-    }
+    open fun setCellData(bottomCellItem: BottomCellItem) {
+        itemView.accessibilityDelegate = object : View.AccessibilityDelegate() {
+            override fun onInitializeAccessibilityNodeInfo(
+                host: View?,
+                info: AccessibilityNodeInfo?
+            ) {
+                super.onInitializeAccessibilityNodeInfo(host, info)
+                print(info)
```

Comment:
Is necessary have this method?

Reply:
Removed in e243856 - good catch it was unnecessary!